pandas/core/generic.py

```diff
@@ -142,6 +143,14 @@
 from pandas.core.array_algos.replace import should_use_regex
 from pandas.core.arrays import ExtensionArray
 from pandas.core.base import PandasObject
+from pandas.core.common import (
```

Comment:
I think we'll need to add `# noqa: PDF018` as a comment here, we can't use the `com` alias